packages/gatsby-source-wordpress/src/output-color.js

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
BTW, chalk is a very popular way of adding color in node that you could use instead of your own module https://www.npmjs.com/package/chalk